### PR TITLE
feat(zoolander):rsweb-5480

### PR DESCRIPTION
**Pull request by amberRrucker**
https://jira.rax.io/browse/RSWEB-5480
For visibility of the tabs/accordion transition.

### CODE REVIEW

**Comment by bighappyface** [comment]
I think we should minify `bootstrap-tabcollapse.js` and push it up to our cloud files container to be used between sites.

**Comment by amberRrucker** [comment]
@bighappyface sounds good to me. where do you want it?

**Comment by bighappyface** [comment]
Go ahead and put it in the `rscom_js` container. Maybe create a `zoolander/derek` directory to put it in?

**Comment by amberRrucker** [comment]
Hey @bighappyface I moved the js file and the fonts should be included properly now @ejchiefs. Let me get to a good place with the tabs styling we talked about yesterday and this PR will fix that font issue.

File: styleguide/_layouts/default.jade [hunk 1]
```diff
@@ -16,6 +16,7 @@ html(lang="en")
 
     link(href="#{ css }", rel="stylesheet")
     link(href="css/main.css", rel="stylesheet")
+    link(href="http://fonts.googleapis.com/css?family=Open+Sans:700,300,600,800,400|Titillium+Web:300,600,400,700|Quicksand:400,300|Montserrat:400,700", rel="stylesheet")
```
Comment (bighappyface) [inline styleguide/_layouts/default.jade]: Let's strip the protocol or change it to https

**Comment by bighappyface** [comment]
+1

**Pull request merged**